### DISCUSSION TITLE
Bar too long

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#101] [Q&A]
Thw bar which shows what tab you are on display the title of the YouTube video sometimes is too big that it pushes remaining icons out of the screen

**Comment by T-Crypt** [comment]
@GeekWolf007 have you adjusted or modified the scaling of your desktop environment in the monitors.conf?

**Comment by sudobrat** [comment]
i have adjusted the scalling it works now on all apps but the only issue is this scaling forced the waybar too small .
rest all is fine

**Reply by T-Crypt** [reply]
@GeekWolf007 There is an open discussion or issue addressing the scaling on the waybar if you take a look, off the top of my head I don't remember the tag!
